src/components/Form/FormField.tsx

```diff
@@ -0,0 +1,147 @@
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+import * as _ from 'lodash'
+
+import { UIComponent, customPropTypes, childrenExist } from '../../lib'
+import { ComponentVariablesInput, ComponentPartStyle } from '../../../types/theme'
+import {
+  Extendable,
+  ReactChildren,
+  ShorthandValue,
+  ShorthandRenderFunction,
+} from '../../../types/utils'
+import Text from '../Text'
+
+export interface IFormFieldProps {
```

Comment:
one prop that I see missing here is, probably, `errorMessage` one (or general `message`) - the text that will be displayed if field value's validation had failed/passed (text below the input field on the picture): 

![image](https://user-images.githubusercontent.com/9024564/46948826-9ee48300-d07f-11e8-857b-2a1f7a151d86.png)

Do we have any plans to support it?

Reply:
Added message prop for the form field component that appears below the input.